01-JS-DOM-APIs/exercise12.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html class="html">
+<head>
+   <link href="normalize.css" rel="stylesheet">
+   <link href="exerciseJS.css" rel="stylesheet">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <meta charset="UTF-8">
+   <title>Excercise 2</title>
+   <script type="text/javascript" src="exercise12.js"> </script>
+   </head>
+<body onload="createMatrix()">
+  <table id="dataTable" class="dataTables">
+    <tr>
+      <th class="tableTH">Firstname</th>
+      <th class="tableTH">Lastname</th>
+      <th class="tableTH">Prefered Country</th>
+    </tr>
+  </table>
+  <button onclick="importMatrixToDOM(matrix)" id="importMatrixButton" type="buttonImportMatrix" name="buttonImportMatrix">Import</button>
```

Comment:
Type `buttonImportMatrix` doesn't exist. This are the current button types http://www.w3schools.com/tags/att_button_type.asp